src/javascripts/main.js

```diff
@@ -2,14 +2,41 @@ import firebase from 'firebase/app';
 import apiKeys from './helpers/apiKeys.json';
 
 import auth from './components/auth/auth';
-import myNavbar from './components/myNavbar/myNavbar';
 import authData from './helpers/data/authData';
+import mushroomList from './components/mushroomList/mushroomList';
+import mycologistList from './components/mycologistList/mycologistList';
+import myNavbar from './components/myNavbar/myNavbar';
 
 import '../styles/main.scss';
 
+const authDiv = $('#auth');
+const forestDiv = $('#forest');
+const hutDiv = $('#hut');
+const logoutButton = $('#navbar-logout-button');
+const singleMycoDiv = $('#single-myco');
+
+const showLoggedInView = () => {
+  authDiv.addClass('hide');
+  forestDiv.removeClass('hide');
+  hutDiv.removeClass('hide');
+  singleMycoDiv.removeClass('hide');
+  logoutButton.removeClass('hide');
+
+  mushroomList.buildForest();
+  mycologistList.buildHut();
+};
+
+const showLoggedOutView = () => {
+  authDiv.removeClass('hide');
+  forestDiv.addClass('hide');
+  hutDiv.addClass('hide');
+  singleMycoDiv.addClass('hide');
+  logoutButton.addClass('hide');
+};
```

Comment:
so @gwynnemeeks any suggestions for a good filename for this functionality? `lazy.js`, while descriptive of me, isn't super descriptive of what the file would actually be used for 😄

Reply:
`showAndTell.js`
`hideAndShow.js`

That's about as creative as my brain can be?  Or, in more seriousness `authClass.js` is probably the more professional title.